packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -1,26 +1,271 @@
 describe('Launchpad: Setup Project', () => {
-  before(() => {
-    cy.scaffoldProject('todos')
-  })
-
   beforeEach(() => {
-    cy.openProject('todos')
-    cy.visitLaunchpad()
+    cy.scaffoldProject('pristine') // not configured
+    cy.scaffoldProject('pristine-with-ct-testing') // component configured
+    cy.scaffoldProject('pristine-with-e2e-testing') // e2e configured
   })
 
+  const verifyWelcomePage = ({ e2eIsConfigured, ctIsConfigured }) => {
+    cy.contains('Welcome to Cypress!').should('be.visible')
+    cy.contains('[data-cy-testingtype="e2e"]', e2eIsConfigured ? 'Configured' : 'Not Configured')
+    cy.contains('[data-cy-testingtype="component"]', ctIsConfigured ? 'Configured' : 'Not Configured')
+  }
+
   it('no initial setup displays welcome page', () => {
+    cy.openProject('pristine')
+    cy.visitLaunchpad()
     cy.contains('Welcome to Cypress!').should('be.visible')
-    cy.contains('E2E Testing').should('be.visible')
-    cy.contains('Component Testing').should('be.visible')
+    verifyWelcomePage({ e2eIsConfigured: false, ctIsConfigured: false })
+  })
+
+  describe('"learn about testing types" modal', () => {
+    beforeEach(() => {
+      cy.openProject('pristine')
+      cy.visitLaunchpad()
+      verifyWelcomePage({ e2eIsConfigured: false, ctIsConfigured: false })
+    })
+
+    it('welcome page has "learn about testing types" link which opens modal', () => {
+      cy.contains('Review the differences').click()
+      cy.get('#app').should('have.attr', 'aria-hidden', 'true')
+      cy.contains('Key Differences').should('be.visible')
```

Comment:
You can get a tighter assertion by doing something like:
```js
cy.findByRole('dialog', { name: 'Key Differences' }.within(() => {
  // assert scoped dialog content here
})
```

It'd be nice to dry up things like the app being hidden when a dialog is open and other common dialog behaviors (like the ever-present "Need help?" link). Custom command maybe? Not necessary to change here but worth some thinking.

```js
cy.getDialog({ name: '...', rootSelector: '#app' }) // asserts aria state common dialog features -> returns dialog element 
.within()
```

Reply:
Good point. I was finding cy.contains would match a model on the dom even if it wasn't open. I'll make the updates.